README.md

```diff
@@ -57,6 +57,13 @@ The Measure Repository Service server supports `Library` and `Measure` resource
 - url
 - version
 
+### Package
+
+The Measure Repository Service server supports the `Library` and `Measure` `$package` operation with the `id` and `identifier` parameters and SHALL parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). These parameters are:
+
+- url
+- version
```

Comment:
Is there a way to make it clearer that users must provide one of either `id`, `url`, or `identifier`? Maybe something like `... users must provide either id, url, or identifier parameters and may provide any additional SHALL parameters specified in the shareable measure repository section...`